src/runner.js

```diff
@@ -17,17 +17,17 @@ const MAX_RETRIES = 3
  * @param  {String} logDir     path to directory to store logs
  * @return {Object}            containing result and detail information of performance test
  */
-export default async function runPerformanceTest (username, accessKey, argv, name, build, logDir, retryCnt = 0) {
+export default async function runPerformanceTest(username, accessKey, argv, name, build, logDir, retryCnt = 0, crmuxdriverVersion = 'stable') {
```

Comment:
Adding `stable` as default is not necessary given that we handle defaults in the params config.